### ISSUE TITLE
When executing lines, also execute necessary imports

### ISSUE THREAD

**Issue by olfMombach** [nileshkhaire/QuickShell#6]
One of the most useful feature in Eclipse is that it imports necessary things for you. However when executing lines the imports are not transferred to jshell. It would be useful to have functionality that takes necessary imports to jshell when executing a line.

**Comment by nileshkhaire** [comment]
Yes this would be useful but finding imports corresponding to lines would be challenging. 

And most of the people are using Jshell to test small code .

But thanks for the idea. Need to think on it .
